client/payment-details/dispute-details/dispute-footer.tsx

```diff
@@ -0,0 +1,187 @@
+				sprintf(
+					isInquiry( dispute )
+						? /* Translators: %s - formatted date, <a> - link to documentation page */
+						  __(
+								'You submitted evidence for this inquiry on %s. The cardholder’s bank is reviewing the case, which can take 120 days or more. You will be alerted when they make their final decision. <a>Learn more</a>.',
+								'woocommerce-payments'
+						  )
+						: /* Translators: %s - formatted date, <a> - link to documentation page */
+						  __(
+								'You submitted evidence for this dispute on %s. The cardholder’s bank is reviewing the case, which can take 60 days or more. You will be alerted when they make their final decision. <a>Learn more about the dispute process</a>.',
+								'woocommerce-payments'
+						  ),
+					submissionDateFormatted
+				),
+				{ a: disputeDocsLinkElement }
+			);
+			break;
+		case 'lost':
+			if ( isAccepted ) {
+				// Lost accepted
+				message = createInterpolateElement(
+					sprintf(
+						/* Translators: %1$s - formatted date, %2$s – the formatted dispute fee amount, <a> - link to documentation page */
+						__(
+							'This dispute was accepted and lost on %1$s. The %2$s fee has been deducted from your account, and the disputed amount returned to the cardholder. <a>Learn more about preventing disputes</a>.',
+							'woocommerce-payments'
+						),
+						closedDateFormatted,
+						disputeFeeFormatted
+					),
+					{ a: disputeDocsLinkElement }
+				);
+			} else if ( isSubmitted ) {
+				// Lost
+				message = createInterpolateElement(
+					sprintf(
+						/* Translators: %1$s - formatted date, %2$s – the formatted dispute fee amount, <a> - link to documentation page */
+						__(
+							'This dispute was lost on %1$s. The %2$s fee has been deducted from your account, and the disputed amount returned to the cardholder. <a>Learn more about preventing disputes</a>.',
+							'woocommerce-payments'
+						),
+						closedDateFormatted,
+						disputeFeeFormatted
+					),
+					{ a: disputeDocsLinkElement }
+				);
+			} else {
+				// Lost no response
+				message = createInterpolateElement(
+					sprintf(
+						/* Translators: %1$s - formatted date, %2$s – the formatted dispute fee amount, <a> - link to documentation page */
+						__(
+							'This dispute was lost on %1$s due to non-response. The %2$s fee has been deducted from your account, and the disputed amount returned to the cardholder. <a>Learn more about preventing disputes</a>.',
+							'woocommerce-payments'
+						),
+						closedDateFormatted,
+						disputeFeeFormatted
+					),
+					{ a: disputeDocsLinkElement }
+				);
+			}
+			break;
+	}
+
+	const handleClick = () => {
+		wcpayTracks.recordEvent(
+			wcpayTracks.events.TRANSACTION_DETAILS_DISPUTE_FOOTER_BUTTON_CLICK,
+			{
+				dispute_status: dispute.status,
+				dispute_submitted: isSubmitted,
+				dispute_accepted: isAccepted,
+			}
+		);
+		if ( isSubmitted ) {
+			const challengeUrl = getAdminUrl( {
```

Comment:
This logic feels over complicated to me – is this inferring that if the dispute is submitted, the button links to view the submitted response? 

If we could tightly couple the button action to the markup for each case it would be simpler IMO.

Where's the button handler that links to the dispute detail screen? (I tested it so I know it works, but I thought I'd see it here somewhere!)

Reply:
> Where's the button handler that links to the dispute detail screen?

I think you mean the button that links to the dispute challenge screen, in which case, this is the handler for that.

I haven't refactored this yet, will take another look tomorrow.

Reply:
> I think you mean the button that links to the dispute challenge screen, in which case, this is the handler for that.

I was confused yesterday, I thought the button did different actions (based on the label/confusion). I see now that the action is the same for all 😁